array-manipulation-3/src/union.js

```diff
@@ -31,3 +31,4 @@ function union(first, second) {
   return uniqueArr;
 
 }
+//
```

Comment:
Having a nested loop in this function should not be necessary. Once you've made your concatenated array named `array`, you should just loop over it and check if `uniqueArray` contains that value. If not, add that value to the array.